Polyhedron/doc/Polyhedron/Concepts/PolyhedronItems_3.h

```diff
@@ -13,9 +13,10 @@ polyhedral surface renames faces to facets.
 
 \cgalRefines{HalfedgeDSItems}
 
-\cgalHasModel `CGAL::Polyhedron_items_3`
-\cgalHasModel `CGAL::Polyhedron_min_items_3`
-\cgalHasModel `CGAL::Polyhedron_items_with_id_3`
+\cgalHasModelsBegin CGAL::Polyhedron_items_3
```

Comment:
```suggestion
\cgalHasModelsBegin
 \cgalHasModels CGAL::Polyhedron_items_3
```

Reply:
The problem noted in the overnight build was that there was a merge conflict and this lead probably to the problem shown. 
(Note the difference `cgalHasModel` versus `cgalHasModels` (so with `s`).
I think the overnight build error should be gone due to the merge I did.

Reply:
That was my first idea as well, but I had problems here regarding an extra empty line in the output due to the fact that we get an extra field (and thus extra unwanted empty space):
![image](https://github.com/CGAL/cgal/assets/5223533/40bcf356-8aa5-471f-857d-45f217a56751)

Here you see my implementation and the suggested change.

With the knowledge I now have, I think it would be possible to define:
```
                         "cgalHasModelsBegin=<dl><dt>@cgalHasModelsHeader</dt><dd>" \
                         "cgalHasModels{1}=<dt></dt><dd>`\1`</dd>" \
                         "cgalHasModels{2}=<dt></dt><dd>`\1,\2`</dd>" \
                         "cgalHasModelsEnd=</dl>" \
```
in case of 
```
CGAL::Surface_mesh_shortest_path_traits<K,P>
```
this would also give the good results.

There are also some long model names like:
```
CGAL::Nonnegative_linear_program_from_iterators<A_it, B_it, R_it, FL_it, L_it, FU_it, U_it, D_it, C_it>
```
but this would use the `cgalHasModels{1}` as fallback.
Actually I'm not a fan of this fallback as it makes things less clear.

The initial idea I had was that the `cgalHasModels` would contain all the models for the list (like with `cgalModels`) and than the fallback would not work anymore.

In the mean time I also created a proposed pull request in doxygen (https://github.com/doxygen/doxygen/pull/10179) that makes it possible to define a separator, so at that moment the fallback would not even be necessary anymore when we use as separator e.g. `;`.

Conclusion  I should probably rework the `cgalHasModels` part, using the `{1}` construct and the "fallback".
(I, probably, will have a look at it during the weekend).